src/configs/strings/en.json

```diff
@@ -870,5 +870,13 @@
   "LINK": {
     "message": "Link",
     "description": "URL leading to another page"
+  },
+  "AN_ERROR_OCCURRED": {
+    "message": "An error occurred",
+    "description": "Generic error message"
+  },
+  "JURISDICTION_LOADING_ERROR": {
+    "message": "An error prevented this jurisdiction from loading.",
+    "description": "Error message displayed when jurisdiction cannot be displayed correctly"
```

Comment:
Let's also be sure to request these translations 🙏

Reply:
How do we do that usually @cKellyDesign ?

Reply:
I think [we can just add them to the spreadsheet here](https://docs.google.com/spreadsheets/d/1ONHpPpuPxNW9_-vP8gTTcrVHTbKTT3WaN-cfdySeMCs/edit?ts=5e0d6060#gid=0) and let @craigappl and/or @samkanga know that there are more display strings to be translated so they can let the client know to provide them.